packages/sdk/contractkit/src/wrappers/StableTokenRegistry.ts

```diff
@@ -0,0 +1,34 @@
+import { string } from 'io-ts'
+import { StableTokenRegistry } from '../generated/StableTokenRegistry'
+import { BaseWrapper, proxyCall } from './BaseWrapper'
+
+interface ContractInstance {
+  concatenatedContracts: string
+  contractLengths: string
+}
+
+export class StableTokenRegistryWrapper extends BaseWrapper<StableTokenRegistry> {
+  owner = proxyCall(this.contract.methods.owner)
+  fiatTickers = proxyCall(this.contract.methods.fiatTickers)
+  stableTokens = proxyCall(this.contract.methods.stableTokens)
+
+  /**
+   * Returns the contatenated contracts and each of their lengths
+   * @return concatenatedContract, ContractLengths
+   */
+  async getContractInstances(): Promise<ContractInstance> {
+    const ret = await Promise.resolve(this.contract.methods.getContractInstances())
+    return {
+      concatenatedContracts: Object.keys(ret)[0],
+      contractLengths: Object.keys(ret)[1],
```

Comment:
I don't think it's needed to return the length here, as in TS you can just query the size